presto-hive/src/main/java/com/facebook/presto/hive/OrcFileWriterFactory.java

```diff
@@ -133,7 +133,7 @@ public OrcFileWriterFactory(
         this.nodeVersion = requireNonNull(nodeVersion, "nodeVersion is null");
         this.hiveStorageTimeZone = requireNonNull(hiveStorageTimeZone, "hiveStorageTimeZone is null");
         this.readStats = requireNonNull(readStats, "stats is null");
-        this.orcWriterOptions = requireNonNull(orcWriterOptions, "orcWriterOptions is null");
+        this.orcWriterOptionsBuilder = requireNonNull(orcFileWriterConfig, "orcFileWriterConfig is null").toOrcWriterOptionsBuilder();
```

Comment:
I would recommend avoiding storing a mutable builder as a class field. It could be error prone. How about storing an immutable instance of the `OrcWriterOptions` and create a builder based on that if needed?

Reply:
Removed the builder.